src/parsing.jl

```diff
@@ -344,42 +345,16 @@ function create_args_vector(args...)
 end
 
 """
-   create_args_vector(arg) -> vec, wrap_byrow
-
-Normalize a single input to a vector of expressions,
-with a `wrap_byrow` flag indicating that the
-expressions should operate by row.
-
-If `arg` is a single `:block`, it is unnested.
-Otherwise, return a single-element array.
-Also removes line numbers.
-
-If `arg` is of the form `@byrow ...`, then
-`wrap_byrow` is returned as `true`.
+   create_args_vector(arg) -> vec, outer_flags
```

Comment:
Why remove the contents of the docstring?

Reply:
I will add a correct docstring.